src/style.css

```diff
@@ -0,0 +1,36 @@
+body {
+  margin: 0;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-position: center center;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars .stars__star {
+  background: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars.stars--1 .stars__star:nth-child(-n+1),
+.stars.stars--2 .stars__star:nth-child(-n+2),
+.stars.stars--3 .stars__star:nth-child(-n+3),
+.stars.stars--4 .stars__star:nth-child(-n+4),
+.stars.stars--5 .stars__star:nth-child(-n+5) {
+  background: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
```

Comment:
There is no need to duplicate the code here, if these properties already exist above
```suggestion
```